gemhardware/include/gem/hw/HwGenericAMC.h

```diff
@@ -15,32 +15,17 @@ namespace gem {
       {
       public:
 
-        /* // FIXME, THIS SHOULD NOT BE HARDCODED, move out of class? */
-        /* static constexpr uint8_t N_GTX = 12; ///< maximum number of GTX links on the GenericAMC */
-
         /**
-         * @struct AMCIPBusCounters
+         * @struct AMCIPBusCounters FIXME UPDATE FOR V3
          * @brief This structure stores retrieved counters related to the GenericAMC IPBus transactions
-         * @var AMCIPBusCounters::OptoHybridStrobe
-         * OptoHybridStrobe is a counter for the number of errors on the tracking data link
-         * @var AMCIPBusCounters::OptoHybridAck
-         * OptoHybridAck is a counter for the number of errors on the trigger data link
-         * @var AMCIPBusCounters::TrackingStrobe
-         * TrackingStrobe is a counter for the number of errors on the tracking data link
-         * @var AMCIPBusCounters::TrackingAck
-         * TrackingAck is a counter for the number of errors on the trigger data link
-         * @var AMCIPBusCounters::CounterStrobe
-         * CounterStrobe is a counter for the number of errors on the tracking data link
-         * @var AMCIPBusCounters::CounterAck
-         * CounterAck is a counter for the number of errors on the trigger data link
          */
         typedef struct AMCIPBusCounters {
```

Comment:
Since you're already changing the lines above and below, please drop the extra `typedef`, this is C++, not C.

Reply:
Future cleanup